src/services/game-logic/Board.ts

```diff
@@ -1,10 +1,11 @@
 import { Square } from '../../models/Square';
 import { Colors } from '../../enums/Colors';
+import { Bishop } from './pieces/Bishop';
 import { Pawn } from './pieces/Pawn';
 import { Piece } from './pieces/Piece';
 
 class Board {
-    public static BOARD_SIZE: number = 8;
+    public static BOARD_SIZE = 8;
```

Comment:
Have you ran `npm install`? This will install newly added plugin, and this line should work with `: number`

Reply:
dodanie tutaj typu to redundancja- typeScript zrobi to za nas bo od razu inicjalizujemy zmienną.